guides/field-projection.md

```diff
@@ -0,0 +1,23 @@
+# Field Projection
+
+Some endpoints support field projection,
```

Comment:
"Some endpoints" ---> are they always/only Lists and Query endpoints?

Reply:
Nope, can apply to any endpoint that returns the entity. It's entirely up to the dev team. It's just a standard on list and query by default, and even then the team has the option of omitting field projection options.